tests/pytorch/test_pytorch_model_export.py

```diff
@@ -1201,3 +1201,22 @@ def forward(self, x, y):
         np.testing.assert_array_almost_equal(
             pyfunc_model.predict(x, {"y": 2}), model(x, 2), decimal=4
         )
+
+
+@pytest.mark.parametrize("scripted_model", [False])
```

Comment:
do we need this? Can we directly use `get_sequential_model`?

Reply:
Yes we can, we don't really need to train the model